main/inc/lib/main_api.lib.php

```diff
@@ -2560,6 +2562,16 @@ function api_is_teacher() {
 }
 
 /**
+ * Checks whether the current user is a invited user
+ * @return boolean
+ */
```

Comment:
I modified the function name from api_is_invitee_user() to api_is_invitee(), because in English, invitee (the ee ending) already assumes it is someone (or something).